subsys/net/ip/net_pkt.c

```diff
@@ -137,13 +129,11 @@ static bool net_pkt_alloc_add(void *alloc_data, bool is_pkt,
 		net_pkt_allocs[i].func_alloc = func;
 		net_pkt_allocs[i].line_alloc = line;
 
-		return true;
+		return;
 	}
-
-	return false;
 }
 
-static bool net_pkt_alloc_del(void *alloc_data, const char *func, int line)
+static void net_pkt_alloc_del(void *alloc_data, const char *func, int line)
```

Comment:
Maybe instead of making this void we actually should log ERR when e.g. alloc entry is not found?

Reply:
I had bool value there earlier but it is quite useless actually as there is nothing caller can do when an error happens.

Reply:
Can report error, so it's extra integrity check. Anyway, approving, thanks.